app/javascript/app/components/emission-pathways/emission-pathways-graph/emission-pathways-graph-selectors.js

```diff
@@ -160,18 +161,19 @@ export const getScenariosOptions = createSelector(
 );
 
 export const getScenariosSelected = createSelector(
-  [getScenariosOptions, getScenario, getModelSelected],
-  (scenarios, scenarioSelected, model) => {
+  [getScenariosOptions, getScenarios, getModelSelected],
+  (scenarios, scenariosSelected, model) => {
     if (!scenarios || !model) return null;
-    if ((!scenarioSelected && !model.scenarios) || scenarioSelected === '') {
+    if ((!scenariosSelected && !model.scenarios) || scenariosSelected === '') {
       return null;
     }
-    if (!scenarioSelected) {
+    if (!scenariosSelected) {
       return scenarios;
     }
-    const scenarioSelectedParsed = scenarioSelected
-      .split(',')
-      .map(s => parseInt(s, 10));
+    let scenarioSelectedParsed = isArray(scenariosSelected)
+      ? scenariosSelected
+      : scenariosSelected.split(',');
+    scenarioSelectedParsed = scenarioSelectedParsed.map(s => parseInt(s, 10));
```

Comment:
Seems weird using a `map` to mutate the mapped object. Wouldn't it make sense to name the first `let scenarioSelectedParsed` differently (e.g. `const selectedScenariosArray`) (as what you're doing is checking if it's an array and converting it to one if it is not) an then `map` over it assigning the new array to `const scenarioSelectedParsed`?